plugins/woocommerce/src/Admin/API/Reports/Categories/Query.php

```diff
@@ -21,7 +21,9 @@
 use Automattic\WooCommerce\Admin\API\Reports\Query as ReportsQuery;
 
 /**
- * API\Reports\Query
+ * API\Reports\Categories\Query
+ *
+ * @deprecated x.x.x Categories\Query class is deprecated, please use GenericQuery or \WC_Object_Query instead.
```

Comment:
There are still some parts that have `x.x.x`, do we need to also update them to `9.3.0`?